Services/CO.CDP.OrganisationInformation.Persistence.Tests/DatabasePersonRepositoryTest.cs

```diff
@@ -71,14 +71,19 @@ public async Task ItUpdatesAnExistingPerson()
     {
         using var repository = PersonRepository();
 
-        var person = GivenPerson();
-        person.Email = "email@email.com";
+        var guid = Guid.NewGuid();
+        var initialEmail = "initial@example.com";
+        var initialDate = DateTime.UtcNow.AddDays(-1);
+        var person = GivenPerson(guid: guid, email: initialEmail);
+
+        person.Email = "updated@example.com";
         repository.Save(person);
 
         var found = await repository.Find(person.Guid);
 
         found.Should().NotBeNull();
-        found!.Email.Should().Be("email@email.com");
```

Comment:
To avoid `!` which usually raises a red flag, we could use `found.As<Person>.Email.Should().Be("email@email.com")`.